proknow-sdk-test/TestEnvironment/templates/config.json

```diff
@@ -0,0 +1,151 @@
+{
+  "hostname": "http://localhost:3005",
+  "subdomain": "pk-test",
+  "security": {
+    "accessKeyId": "proknow",
+    "secretAccessKey": "password",
+    "certificate": "support/certs/localhost.crt",
+    "key": "support/certs/localhost.key"
+  },
+  "maxUploadFileSize": 50000000,
+  "cache": "/var/opt/proknow",
+  "storage": {
+    "type": "s3",
+    "name": "atlas",
+    "config": {
+      "accessKeyId": "S3_ACCESS_KEY_ID",
+      "secretAccessKey": "S3_ACCESS_KEY_SECRET",
+      "region": "us-east-1",
+      "sslEnabled": false
+    },
+    "client": {
+      "endpoint": "http://minio:9000",
+      "s3ForcePathStyle": true,
+      "sslEnabled": false,
+      "bucket": "atlas-test"
+    }
+  },
+  "audit": {
+    "postgres": true,
+    "archiver": {
+      "storage": {
+        "type": "s3",
+        "name": "atlas",
+        "config": {
+          "accessKeyId": "S3_ACCESS_KEY_ID",
+          "secretAccessKey": "S3_ACCESS_KEY_SECRET",
+          "region": "us-east-1",
+          "sslEnabled": false
+        },
+        "client": {
+          "endpoint": "http://minio:9000",
+          "s3ForcePathStyle": true,
+          "sslEnabled": false,
+          "bucket": "atlas-test"
+        }
+      }
+    }
+  },
+  "database": {
+    "type": "postgres",
+    "schema": "/opt/proknow/env",
+    "host": "postgres",
+    "port": 5432,
+    "username": "postgres",
+    "database": "atlas_test"
+  },
+  "identity": {
+    "type": "auth0",
+    "account": "proknow-dev",
+    "domain": "https://auth.proknow-dev.com",
+    "jqksUri": "https://auth.proknow-dev.com/.well-known/jwks.json",
+    "issuer": "https://auth.proknow-dev.com/",
+    "audience": "https://app.proknow.com/api/",
+    "clientId": "AUTH0_CLIENT_ID",
+    "clientSecret": "AUTH0_CLIENT_SECRET",
+    "connection": "Username-Password-Authentication",
+    "namespace": "pk-qpUj"
+  },
```

Comment:
Is the user database being cleaned up after each run? I couldn't find evidence of this. I would do one of the following (I would prefer 2):

1. Delete the organization at the end of the run. This should delete the Auth0 resources that were created. Note that if the build is canceled (due to timeout or even manually by a developer), this will not run, and we will have an orphaned user database in Auth0. :(
2. Figure out how to use Keycloak instead. This can take advantage of the fact that the Keycloak database will be cleaned up with the rest of the resources (PostgreSQL databases, Minio containers, etc.), so it doesn't matter whether you clean up the organization.